### PR TITLE
Add Docker environment & web demo

### PR DESCRIPTION
**Pull request by ericguizzo**
Hey @Wendison! 👋 

I really liked your implementation and it works very well with any kind of voice! Really funny :) 

This pull request makes it possible to run your model inside a Docker environment, which makes it easier for other people to run it.  We're using an open source tool called [Cog](https://github.com/replicate/cog) to make this process easier.

This also means we can make a web page where other people can try out your model! View it here: https://replicate.ai/wendison/vqmivc

[Claim your page here](https://replicate.ai/wendison/vqmivc?published=1) so you can edit it, and we'll feature it on our website and tweet about it too.

In case you're wondering who I am, I'm from [Replicate](https://replicate.ai/home), where we're trying to make machine learning reproducible. We got frustrated that we couldn't run all the really interesting ML work being done. So, we're going round implementing models we like. 😊

### CODE REVIEW

File: predict.py [hunk 1]
```diff
@@ -1,27 +1,147 @@
...
+        f0[nonzeros_indices]
+    )  # for f0(Hz), lf0 > 0 when f0 != 0
+    mean, std = np.mean(lf0[nonzeros_indices]), np.std(lf0[nonzeros_indices])
+    lf0[nonzeros_indices] = (lf0[nonzeros_indices] - mean) / (std + 1e-8)
+    return mel, lf0
 
 
 class Predictor(cog.Predictor):
     def setup(self):
-        """Load the fastspeech and melgan models"""
```
Comment (Wendison) [inline predict.py]: I think this line should be deleted

**Comment by Wendison** [comment]
Thanks for your work! Do you mind if I mention your work in README?

**Comment by ericguizzo** [comment]
Hi Wendison, thanks for merging it!! 
Sure! It would be great if you mention the replicate demo also in the README :)

**Comment by Wendison** [comment]
Hi @ericguizzo , I have added your work and replicate demo in README. Besides, may I know if you could add some details on how to use Cog for development (including installation, model training & testing, etc.), that would be very helpful for others! Thank you!

**Comment by andreasjansson** [comment]
Hi @Wendison! I just sent you an email with a beta invite to Replicate. There's documentation about how and why Cog works on the Replicate home page (https://replicate.ai/home), and more detailed Cog documentation here: https://github.com/replicate/cog

Feel free to ask us questions any time in Discord! https://discord.gg/QmzJApGjyE

**Pull request merged**